app/i18n/ru/conf.php

```diff
@@ -178,7 +178,7 @@
 			'view' => 'Вид',
 		),
 		'hide_read_feeds' => 'Скрывать категории и ленты без непрочитанных статей (не работает с «Показывать все статьи»)',
-		'img_with_lazyload' => 'Использовать режим "ленивой загрузки" для загрузки картинок',
+		'img_with_lazyload' => 'Использовать режим «ленивой загрузки» для загрузки картинок',
```

Comment:
Is there a reason why those are different?

Reply:
I picked the same convention than a few lines higher up, but I do not know what is most usual

Reply:
But on line 117, you've used the typographic quotes.

Reply:
Yes, I believe it is a bit like in French with a few possibilities. We could ask if we find anyone mastering Russian typography :-)